tests/lib/tools/os.query

```diff
@@ -4,7 +4,7 @@ show_help() {
     echo "usage: is-core, is-classic"
     echo "       is-core16, is-core18, is-core20"
     echo "       is-trusty, is-xenial, is-bionic, is-focal"
-    echo "       is-debian, is-fedora, is-amazon-linux, is-arch, is-centos, is-opensuse"
+    echo "       is-ubuntu, is-debian, is-fedora, is-amazon-linux, is-arch, is-centos, is-opensuse"
```

Comment:
mentiones is-arch, but the function name  is `is_arch_linux`, so I suppose it should be `is-arch-linux` here?

Reply:
yes, fixed